.github/probots.yml

```diff
@@ -0,0 +1,2 @@
+enabled:
+  - cla
```

Comment:
what are the implications of this? Do we need to enable or change any checks in our github repo settings?